functions/admin-pages.php

```diff
@@ -83,19 +83,38 @@ function gnt_settings_page_content() {
  */
 function gnt_info_page_content() {
 	?><div class="wrap">
+
 		<h1><?php esc_attr_e( 'Information', 'get-notified' ); ?></h1>
+
+		<p>Get Notified sends you notifications when certain events happen on your site.</p>
```

Comment:
All text strings must be translated and escaped with `esc_html_e` or `esc_html__` depending on how you echo it.  Please address the other text strings in this file that are supposed to be html escaped but are attribute escaped instead.